docs/pages/enroll-resources/kubernetes-access/kubernetes-access.mdx

```diff
@@ -1,31 +1,97 @@
+description: Protect Kubernetes clusters with granular role-based access controls at the cluster or resource level, audit logging, and session recording.
+template: doc-page
 tags:
  - zero-trust
  - infrastructure-identity
 ---
 
-The guides in this section explain how to protect Kubernetes clusters with
-Teleport.
+import DocHero, { DocHeroProps } from "@site/src/components/Pages/Landing/DocHero";
+import Resources from "@site/src/components/Pages/Homepage/Resources";
+import UseCasesList from "@site/src/components/Pages/Landing/UseCasesList";
 
-## Getting started
+import noteSvg from "@site/src/components/Icon/svg/note2.svg";
+import questionSvg from "@site/src/components/Icon/svg/question2.svg";
+import identificationBadgeSvg from "@site/src/components/Icon/svg/identification-badge.svg";
+import listChecksSvg from "@site/src/components/Icon/svg/list-checks.svg";
+import kubernetesClustersSvg from "@site/src/components/Icon/teleport-svg/kubernetes-clusters.svg";
+import iamSvg from "@site/src/components/Icon/svg/awsIdentity.svg";
+import kubernetesSvg from "@site/src/components/Icon/svg/kubernetes2.svg";
 
-- [Introduction to Enrolling Kubernetes Clusters](./introduction.mdx): Learn how Teleport can protect your Kubernetes clusters with RBAC, audit logging, and more.
-- [Enroll a Kubernetes Cluster](./getting-started.mdx): Demonstrates how to enroll a Kubernetes cluster as a resource protected by Teleport.
+{/*vale messaging.protocol-products = NO*/}
+<DocHero
+  title="Enroll your first Kubernetes cluster"
+  youtubeVideoId="rQiloHM2oRQ"
+  ctaLinks={[
+    {
+      title: "Get started with server access",
+      href: "./getting-started/",
+      variant: "secondary",
+      arrow: true,
+    },
+  ]}
+  links={[
+    {
+      title: "Kubernetes Access overview",
+      description: "Securely access Kubernetes cluster with SSO, RBAC and audit logging.",
+      href: "./introduction/",
+      icon: noteSvg,
+    }, 
+    {
+      title: "Troubleshooting",
+      description: "Common Kubernetes access issues and their solutions.",
+      href: "./troubleshooting/",
+      icon: questionSvg,
+    },
+  ]}
+>
+{/*vale messaging.protocol-products = YES*/}
+Log in with SSO and switch between Kubernetes clusters without additional logins. Manage access within clusters with granular RBAC down to specific cluster resources, and achieve compliance needs with session recordings and detailed audit logs.
+</DocHero>
 
-## Guides
+<Resources
+  title="Access controls"
+  variant="doc"
+  desktopColumnsCount={2}
+  resources={[
+    {
+      title: "RBAC for Kubernetes",
+      description: "Configure role-based access control (RBAC) for Kubernetes clusters, groups, users, and resources.",
+      iconComponent: identificationBadgeSvg,
+      href: "./controls/"
+    },
+    {
+      title: "Configure and set up",
+      description: "Enforce RBAC by intercepting user requests and their actions. And configure these roles for your connected clusters.",
```

Comment:
Both of these descriptions kind of sound the same to me.

Can we think of some updated wording that would help highlight the difference between these two links?